config.c

```diff
@@ -13,14 +13,33 @@
 #include "criteria.h"
 #include "types.h"
 
+static struct mako_surface_config *create_surface(struct mako_config *config) {
```

Comment:
This should be called `create_surface_config` to match the struct, especially since a `mako_surface` is a different type.